app/views/documents/show/_requirements.html.erb

```diff
@@ -1,13 +1,15 @@
 <% issue_params = {
   style: "summary",
-  link_options: {
-    title: { href: "#content" },
-    summary: { href: "#content" },
-    body: { href: "#content" },
-    change_note: { href: "#content" },
-    alt_text: { href: "#lead-image" },
-    caption: { href: "#lead-image" },
-    topics: { href: "#topics" }
+  hrefs: {
+    title: edit_document_path(anchor: "title"),
+    summary: edit_document_path(anchor: "summary"),
+    body: edit_document_path(anchor: "body"),
+    change_note: edit_document_path(anchor: "changenote"),
+    #Come back to this at some point so we could link directly to the image?
+    alt_text: images_path(@edition.document, anchor:"imagelist"),
+    caption: images_path(@edition.document, anchor:"imagelist"),
+    #--
+    topics: update_topics_path,
```

Comment:
I notice that this one and the `edit_documents_path` don't receive document as an argument whereas `images_path` does. I assume the ones with a document are working based on some form of magic context in Rails.

I think it'd be better if these consistently took the document argument as that seems to be the most common precedent in the app.